dg/media/app_dashboards/js/training_dashboard.js

```diff
@@ -353,34 +524,55 @@ function plot_trainerwise_data(trainer_list, mediator_list) {
             trainer_pass_perc_dict['data'][i] = parseFloat(perc.toFixed(2));
         }
 
-        trainer_scores_dict.push(perc_score_dict);
         trainer_scores_dict.push(avg_score_dict);
-        trainer_trainings_mediators_dict.push(trainer_trainings_dict);
+        
         trainer_trainings_mediators_dict.push(trainer_mediators_dict);
         trainer_trainings_mediators_dict.push(trainer_mediators_pass_dict);
-        trainer_trainings_mediators_dict.push(trainer_pass_perc_dict);
 
-        plot_dual_axis_chart($("#trainer_mediator_data"), x_axis, trainer_scores_dict, "Average Scores per Participant", "Percent Answered Correctly", "", "%");
-        plot_multiple_axis_chart($("#trainer_training_data"), x_axis, trainer_trainings_mediators_dict, "Trainings", "Mediators", "% Mediators above 70%", "", "", "%");
+        plot_dual_axis_chart($("#trainer_mediator_data"), x_axis, trainer_scores_dict, "Average Scores per Participant", "", "", "%","");
+        plot_multiple_axis_chart($("#trainer_training_data"), x_axis, trainer_trainings_mediators_dict, "Mediators Trained", "", "", "", "", "%", trainer_trainings_dict, "", "Labels represent : Total Trainings","");
     }
 }
 
 function plot_questionwise_data(data_json, assessment_ids) {
-    if (data_json.length == 0) {
-        document.getElementById('question_mediator_data').innerHTML = 'No data for this Assessment!'
+    question_data_length = data_json.length;
+    //Question Array length to store Distinct question : whether it is available in different languages.
+    question_array_length = 0;
+    // offset represent for same Questions in different Languages : 15, if data available in both english and hindi.
+    question_offset = 0;
+    if (question_data_length == 0) {
+        plot_multiple_axis_chart($("#question_mediator_data"), [], {}, "No. of Mediators", "", "","","", "%", {}, "%", "","No data for this assessment");
     } else {
         var x_axis = [];
         var question_dict = [];
+        var question_mediators_dict = {};
+        var question_mediators_passed_dict = {};
+        var question_percent_dict = {};
+
+        question_mediators_dict['pointPadding'] = -0.2;
+        question_mediators_dict['pointPlacement'] = 0.125;
 
+        question_mediators_passed_dict['pointPadding'] = 0.2;
+        question_mediators_passed_dict['pointPlacement'] = -0.168;
+
+        question_mediators_dict['color'] = 'rgba(248,161,63,1)';
+        question_percent_dict['color'] = '#000000';
+        question_mediators_passed_dict['color'] = 'rgba(186,60,61,.9)';
+
+        // For Peeco Seekho, Assessment Id = 1
         if (assessment_ids[0] == 1) {
-            var question_mediators_dict = {};
-            var question_mediators_passed_dict = {};
-            var question_percent_dict = {};
 
+            // For Peeco Seekho, total distinct questions = 15
+
+            question_array_length = 15;
```

Comment:
can we send number of distinct questions for an assessment in the api call. It will be helpful when documentation assessment is added. Then we could merge the computation part for both.